modules/Bio/EnsEMBL/Compara/RunnableDB/MoreAppropriateCollectionToQuery.pm

```diff
@@ -0,0 +1,115 @@
+=head1 LICENSE
+
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::RunnableDB::MoreAppropriateCollectionToQuery
+
+=head1 DESCRIPTION
+
+Runnable to parse the record of query genomes to check and pass on
+for potential update with new taxonomic based reference comparison
+
+=cut
+
+package Bio::EnsEMBL::Compara::RunnableDB::MoreAppropriateCollectionToQuery;
+
+use warnings;
+use strict;
+use List::MoreUtils qw/ uniq /;
+
+use base ('Bio::EnsEMBL::Compara::RunnableDB::BaseRunnable');
+
+sub param_defaults {
+    return {
+        'queries_to_update_file'  => 'queries_to_update.txt',
```

Comment:
Do these filenames need to be hardened-code? Is it just a temp file?

Reply:
This is a parameter default, so it's just to provide a default file to write to if none is specified - which just means less to configure - similar to the report updates from the metadata when we prepare the master database for release in the main site